src/Params.ml

```diff
@@ -96,6 +96,26 @@ let get_record_type (c : Conf.t) ~wrap_record =
       ; break_after= break space (-2)
       ; docked_after= fmt "}" }
 
+type record_expr = {box: Fmt.t -> Fmt.t; sep_before: Fmt.t; sep_after: Fmt.t}
+
+let get_record_expr (c : Conf.t) ~wrap_record =
+  match c.break_separators with
+  | `Before ->
+      { box= (fun k -> hvbox 0 (wrap_record c k))
```

Comment:
I think `wrap_record` should be moved in this module (eg. `type record_pat = { wrap : Fmt.t -> Fmt.t }`)

Reply:
What I meant is that `wrap_record` is used in `Fmt_ast.ml` for record patterns and inline records. So it could be moved by implementing `get_record_pat` and `get_inline_record` (that would only have the `wrap` function) in `Params`. What do you think?

Reply:
I moved the `wrap_` functions to Params.